mucommander-format-xz/build.gradle

```diff
@@ -0,0 +1,29 @@
+// In this section you declare where to find the dependencies of your project
+repositories.jcenter()
+
+dependencies {
+    api project(':mucommander-commons-file')
+
+    implementation 'org.osgi:osgi.core:7.0.0'
+    implementation 'org.tukaani:xz:1.9'
```

Comment:
any reason not to use the wrappers in apache commons-compress (that we already use) instead?

Reply:
The XZ support in commons-compress is a short wrapper around xz-java (and requires it to work) so I didn't see much benefit in using it and went with using xz-java directly. Let me know if you feel otherwise.

Reply:
I took a deeper look and saw that xz-java is not embedded in commons-compress as I thought but is rather set as an optional dependency of commons-compress. I wanted to avoid having the xz-java classes twice but as it's not the case, that's fine